app/models/responsible_body.rb

```diff
@@ -251,6 +251,10 @@ def address
     [address_1, address_2, address_3, town, postcode].reject(&:blank?).join(', ')
   end
 
+  def has_completed_extra_mobile_data_requests?
```

Comment:
Doesn't seem like this + the one added in `school.rb` (below) are being used anywhere?

Reply:
Good spot, it got dropped from the refactoring cycle, same for the models/school.rb, I'll remove it